FreeRTOS-Plus/Demo/FreeRTOS-IoT-Libraries-LTS-Beta2/mqtt/common/main.c

```diff
@@ -45,9 +45,13 @@
 #include "FreeRTOS_IP.h"
 #include "FreeRTOS_Sockets.h"
 
-/* Demo app includes. */
+/* Demo logging includes. */
 #include "demo_logging.h"
```

Comment:
Is this include required or should the `demo_config.h` include this header file?

Reply:
This include is required to call the logging init, otherwise a compile time warning is observed.

Reply:
logging init is called only from main.c, I think that way it is better to be only included in here.